FloconDesktop/composeApp/src/commonMain/kotlin/io/github/openflocon/flocondesktop/features/dashboard/model/DashboardItemViewState.kt

```diff
@@ -1,9 +1,12 @@
 package io.github.openflocon.flocondesktop.features.dashboard.model
 
 import androidx.compose.ui.graphics.Color
+import io.github.openflocon.domain.dashboard.models.ContainerConfigDomainModel
+import io.github.openflocon.domain.dashboard.models.SectionContainerConfigDomainModel
 
 data class DashboardItemViewState(
-    val sectionName: String,
+    val containerName: String,
+    val containerConfig: ContainerConfigDomainModel,
```

Comment:
ui models should be independant of the domain models, 
here : just create an ui model that exposes what you need on the composable

Reply:
This would mean almost a straight copy of the ContainerConfig again, that is already there now for data, domain and entity. If this is what you want I'll update it but just checking, personally I'd say these models would be good to use.

Reply:
Alright, it will still be the same sealed class since thats all the info it needs _(type, formId and submitTest, last two only in case of form)_.
I'll have some time tomorrow to fix this 👌

Reply:
Created the separate ui model 👌